### PR TITLE
New Speakers Section!

### LINKED ISSUE #47: New Speakers Section
The speakers section will contain the following content about each speaker
- Name
- Image
- Social links (twitter, personal website, & linkedin if applicable)
- Title (and company)

image

The content will be stored in a JSON, with references to each image (with are stored in an assets subfolder).

### PR DESCRIPTION
**Pull request by draymond63**
This represents the speakers page, and the subcomponent that also appears on the home page. It closes #47 
image

A few things that would be great if you could take a look at:
- There's some funky JS to limit the number of rows shown (used for the home page section). Is there a better way to do this? Right now I scan across each item's vertical offset, counting the number of items are in each row and then hiding the ones that shouldn't be shown. Probably some way to do it better. Maybe we could get away with just showing a few number of speakers period? This could cause multiple rows, but could be good on mobile (as it wouldn't limit it to one speaker like it does right now), but would cause vacancies in rows.

- Should the background colors be randomized? Right now it's just ordered based on the list it's kept in

- Is there a better way to position the router-link in the header? I feel like using margins isn't very future-proof but it's what I came up with.

Thanks everyone!

### CODE REVIEW

File: src/components/general/Speakers.vue [hunk 1]
```diff
@@ -1,32 +1,172 @@
...
-	<router-link v-if="limit" to="/speakers">View all speakers ></router-link>
+	<div class="header">
+		<h2>Take a Look at CUTC 2020</h2>
+		<router-link v-if="limit" to="/speakers" class="view-more">View all speakers ></router-link>
+	</div>
+
+	<div class="grid" ref="grid">
+		<div class="tile" v-for="(speaker, i) in speakers" :key="speaker">
+			<!-- PHOTO -->
+			<div class="image" :style="{'display': showTile(i)}">
```
Comment (sanaasy) [inline src/components/general/Speakers.vue:11]: just for my own learning could you explain what `:style` does?

File: src/components/general/Speakers.vue [hunk 2]
```diff
@@ -1,32 +1,172 @@
...
-a {
+#speakers {
+	--icon-width: 1.5rem;
+	--tile-radius: 1rem;
+}
+
+.view-more {
 	color: var(--main-color);
 	text-decoration: underline;
+	margin: 1.3em;
```
Comment (sanaasy) [inline src/components/general/Speakers.vue:111]: maybe you could use `float: right;` here?

**Review by sanaasy — COMMENTED** [review]
I didn't have much to comment on but I think what you have in `limitRows()` might be good for now. If I come up with anything by our next meeting, I'll definitely bring it up! As for the colours, it would be cool to randomize them so they show a different colour each time. Overall the section looks great!!! 🥇

File: src/components/general/Speakers.vue (hunk 1 above)
Comment (draymond63) [reply src/components/general/Speakers.vue:11]: Style is like css classes, but makes it so you don't have to create a class. The colon makes it dynamic (not just a string)

**Pull request merged**